addons/info/src/components/PropTable.js

```diff
@@ -82,6 +82,22 @@ const propsFromPropTypes = type => {
   return props;
 };
 
+export const multiLineText = text => {
+  if (!text) return '';
```

Comment:
Does anyone have a preference here to return `null` and therefore render `<td/>` versus an empty string(`''`) and `<td></td>`

Reply:
I think I'd prefer the short version

Reply:
👍